test/common/ssl/test_data/README.md

```diff
@@ -2,7 +2,8 @@
 There are 10 identities:
 - **CA**: Certificate Authority for **No SAN**, **SAN With URI** and **SAN With
   DNS**. It has the self-signed certificate *ca_cert.pem*. *ca_key.pem* is its
-  private key.
+  private key.  Additionally, we create a CRL for this CA (*ca_cert.crl*) that
```

Comment:
One space should be enough for everybody ;)

Reply:
FWIW, this also breaks **ci/circleci: format** check.

Reply:
Aaaaaaagh 😛 

Okay, fixed.